### PR TITLE
Dropped support for GDAL 1.9 and 1.10.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: tests/gis_tests/gdal_tests/test_raster.py [hunk 1]
```diff
@@ -378,37 +371,36 @@ def test_compressed_file_based_raster_creation(self):
         compressed = self.rs.warp({'papsz_options': {'compress': 'packbits'}, 'name': rstfile.name})
         # Check physically if compression worked.
         self.assertLess(os.path.getsize(compressed.name), os.path.getsize(self.rs.name))
-        if GDAL_VERSION > (1, 11):
```
Comment (claudep) [inline tests/gis_tests/gdal_tests/test_raster.py]: This should be kept, as GDAL 1.11 is still supported!

Comment (timgraham) [reply tests/gis_tests/gdal_tests/test_raster.py]: It's just unindented.

Comment (claudep) [reply tests/gis_tests/gdal_tests/test_raster.py]: But with GDAL 1.11, the following lines will execute (note `>` and not `>=`).

Comment (timgraham) [reply tests/gis_tests/gdal_tests/test_raster.py]: Oh, I assumed the condition was a typo for `>=` (unless there's some relevant bug fix in 1.11.1). I think it's unlikely that someone will run the tests with GDAL 1.11.0 at this point but if you prefer to keep the condition, it's probably not worth further investigation.

**Comment by claudep** [comment]
What about decommissioning  the pull-requests-trusty and replace it by pull-request-xenial ? That would also clarify the GDAL 1.11 support question.

**Comment by timgraham** [comment]
Sure, I updated the PR builders.

**Comment by claudep** [comment]
Thanks, I guess https://code.djangoproject.com/wiki/Jenkins should be updated accordingly.

**Pull request merged**